DICOM/src/Microsoft.Health.DeID.SharedLib/Settings/DateShiftSetting.cs

```diff
@@ -3,6 +3,8 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
+using Microsoft.Health.Dicom.DeID.SharedLib.Exceptions;
+
 namespace Microsoft.Health.Dicom.DeID.SharedLib.Settings
 {
     public class DateShiftSetting
```

Comment:
A little confused: Shouldn't DateShiftKeyPrefix be DateShiftScope?
In sample configuration, it's like this:
```
"dateShift": {
            "dateShiftKey": "123",
            "dateShiftScope": "SeriesInstance",
            "dateShiftRange": 50
        },
```

Reply:
for shared lib, the setting names "Anonymizer setting", and it should be dateShiftKeyPrefix.
DateShiftScope is the concept in DICOM, and given in config file's rule setting.
dateShiftKeyPrefix is calculated from DateShiftScope